### PR TITLE
other(localnet): replace nodejs with nginx

### LINKED ISSUE #658: Replace nodejs with nginx in localnet explorer
In a future PR, we can replace nodejs with nginx and save about 200MB+.

_Originally posted by @abukosek in https://github.com/oasisprotocol/oasis-web3-gateway/issues/653#issuecomment-2498007081_

### PR DESCRIPTION
**Pull request by ptrus**
Fixes: https://github.com/oasisprotocol/oasis-web3-gateway/issues/658

Reduces sapphire image size from ~970MB to ~793MB.

### CODE REVIEW

**Review by abukosek — APPROVED** [review]
Awesome!

**Comment by matevz** [comment]
Could we also replace envoy with nginx?

File: docker/common/start.sh [hunk 1]
```diff
@@ -15,6 +15,7 @@
 # - OASIS_NEXUS_BINARY: path to oasis-nexus binary
 # - OASIS_NEXUS_CONFIG_FILE: path to oasis-nexus config file. Required if OASIS_NEXUS_BINARY is provided
 # - OASIS_EXPLORER_DIR: path to explorer (nexus frontend) directory
+# - OASIS_EXPLORER_NGINX: path to explorer nginx config file.
```
Comment (matevz) [inline docker/common/start.sh]: I would prefer the longer naming for consistency.

```suggestion
# - OASIS_EXPLORER_NGINX_CONFIG_FILE: path to explorer nginx config file.
```

**Comment by ptrus** [comment]
> Could we also replace envoy with nginx?

Not if we want to support grpc-web

**Review by kostko — APPROVED** [review]

File: docker/common/start.sh (hunk 1 above)
Comment (ptrus) [reply docker/common/start.sh]: 👍  Updated

**Pull request merged**